### DISCUSSION TITLE
Información detallada sobre como funciona la aplicación Syspass

### DISCUSSION THREAD

**Discussion by Dan00127** [nuxsmin/sysPass#1651] [Q&A] [answered]
Buen día, quisiera saber si me podrian ayudar facilitándome algún tutorial  donde se especifique toda la funcionalidad de la aplicación syspass, como se crean usuarios, grupos, perfiles, en que orden, como se comparten las contraseñas con las demás cuentas de usuarios que se vayan a crear, todo sobre ella, es que la voy a implementar en la empresa y quisiera saber como funciona todo para empezar mi proyecto.

Gracias espero su pronta respuesta y ojala me puedan ayudar.

**Comment by nuxsmin** [comment, marked as answer]
Hola, existe un portal con la documentación relativa a sysPass (https://syspass-doc.readthedocs.io/en/3.1/) y además la mayoría de las acciones tiene una ayuda contextual.

Saludos!

**Comment by Dan00127** [comment]
Hola, Gracias por responder, si he visto el documento pero mayormente
hablan de lo técnico osea como para que sirve los grupos, los perfiles más
no como se crean cómo se implementa en una empresa pero bueno si usted me
dice que hay ayudas contextuales pues no debe ser tan difícil su uso.
De todas maneras puedo seguir contactandome con usted por si se me
dificulta algo a medida que la implementó? seria de gran ayuda.

Gracias y quedo atenta a su respuesta

**Reply by nuxsmin** [reply]
Sin problema, si tiene alguna duda/incidencia al respecto puede preguntar en este apartado.

**Comment by t1c0p4n** [comment]
Hola, actualmente tengo la versión 3.1 (310.19081801). ¿Dónde encuentro información sobre el proceso de actualización?

**Reply by nuxsmin** [reply]
Buenas, las actualzaciones son relativamente sencillas, pero hay algunos pasos recomendados aquí: https://syspass-doc.readthedocs.io/en/3.1/updating/index.html

**Comment by valmsaa** [comment]
Buenas tardes.
Tengo dos máquinas virtuales en las que instalé la versión de sysPass 3.1.1, en una de ellas he actualizado a la versión 3.2.1 sin problema, al ir a actualizar la otra, vi que había sacado una nueva versión y realicé el mismo procedimiento  para dejarla actualizada 3.2.2, pero en este caso no reconoce las credenciales, restauré la máquina y volví a probar y no consigo que funcione. No sé si puede ser porque utilicé el archivo vendor de la versión 3.2.1, porque para la 3.2.2, no estaba disponible la descarga. Si pudiera indicarme qué puede haber fallado.
Saludos.
